src/Framework/TestCase.php

```diff
@@ -2034,6 +2122,15 @@ private function checkRequirements(): void
         );
 
         if (!empty($missingRequirements)) {
+            Event\Registry::emitter()->testSkippedDueToUnsatisfiedRequirements(
+                get_class($this),
```

Comment:
@sebastianbergmann @theseer 

Shall we use `static::class` here instead?

Reply:
Seems to work, go for it :)

Reply:
👍